packages/flutter/lib/src/material/time.dart

```diff
@@ -19,7 +19,26 @@ enum DayPeriod {
 /// A value representing a time during the day, independent of the date that
 /// day might fall on or the time zone.
 ///
-/// The time is represented by [hour] and [minute] pair.
+/// The time is represented by [hour] and [minute] pair. Once created, both
+/// values can not be changed.
+///
+/// You can create TimeOfDay using the constructor which requires both hour and
+/// minute or using [DateTime] object.
+/// Note that hours are specified between 0 and 23, as in a 24-hour clock.
+/// For example:
+///
+/// ```
+/// var now = TimeOfDay.now();
+/// var releaseTime = TimeOfDay(hour: 15, minute: 0); // 3:00pm
+/// var roomBooked = TimeOfDay.fromDateTime(DateTime.parse('2018-10-20 16:30:04Z')); // 4:30pm
+/// ```
+///
+/// You can use properties to get
```

Comment:
This part doesn't really seem to document anything specific to this class? Maybe remove?

Reply:
Removed.